addons/website/static/src/scss/website.scss

```
@@ -732,158 +732,116 @@ table.table_desc tr td {
+                transition: opacity .6s ease, transform .6s ease; // $carousel-indicator-transition values
+
+                // Increases the click area for easier navigation.
+                &:before {
+                    position: absolute;
+                    inset: var(--o-carousel-indicators-spacer);
+                    display: block;
+                    content: '';
+                }
 
-        li:hover:not(.active) {
-            background-color: rgba(255,255,255,.8);
-        }
-    }
+                &:hover {
+                    opacity: 1;
+                }
 
-    // Default
-    &.s_carousel_default {
-        // Controls - chevron
-        .carousel-control-prev-icon:before { content: "\f053" #{"/*rtl:'\f054'*/"}; margin-left: 1.5rem; }
-        .carousel-control-next-icon:before { content: "\f054" #{"/*rtl:'\f053'*/"}; margin-right: 1.5rem; }
-        .carousel-control-prev-icon:before,
-        .carousel-control-next-icon:before {
-            background-color: rgba(0,0,0,0);
-            @include font-size(2rem);
-            color: #fff;
-            text-shadow: $box-shadow-sm;
-        }
-        // Indicators
-        .carousel-indicators li {
-            height: .6rem;
-            margin-bottom: .5rem;
-            border: 0;
-            border-radius: $border-radius-sm;
-            box-shadow: $box-shadow-sm;
-        }
-    }
-
-    // Border
-    &.s_carousel_bordered {
-        border: 2rem solid rgba(0,0,0,0);
-        @include media-breakpoint-down(md) {
-            border: 0.5rem solid rgba(0,0,0,0);
-        }
-        // Controls - caret
-        .carousel-control-prev-icon:before { content: "\f0d9"; }
-        .carousel-control-next-icon:before { content: "\f0da"; }
-        .carousel-control-prev-icon:before,
-        .carousel-control-next-icon:before {
-            width: 2rem;
-            height: 6rem;
-            line-height: 6rem;
-            @include font-size(1.5rem);
-        }
-        // Indicators
-        .carousel-indicators li {
-            width: 3rem;
-            height: 1rem;
-        }
-    }
-
-    // Circle
-    &.s_carousel_rounded {
-        // Container
-        // .carousel-inner {
-            // border-top-left-radius: 10rem;
-            // border-bottom-right-radius: 10rem;
-        // }
-        // Controls - arrow
-        .carousel-control-prev { margin-left: 1.5rem; }
-        .carousel-control-next { margin-right: 1.5rem; }
-        .carousel-control-prev-icon:before { content: "\f060"; }
-        .carousel-control-next-icon:before { content: "\f061"; }
-        .carousel-control-prev-icon:before,
-        .carousel-control-next-icon:before {
-            width: 4rem;
-            height: 4rem;
-            line-height: 4rem;
-            border-radius: 50%;
-            @include font-size(1.25rem);
+                &.active {
+                    --o-carousel-indicators-spacer: #{$carousel-indicator-spacer * -1};
+                    transform: scale(1);
```

convention: empty line between CSS variables definition and regular CSS directives  😉
```suggestion
                    --o-carousel-indicators-spacer: #{$carousel-indicator-spacer * -1};
                    
                    transform: scale(1);
```